index.js

```diff
@@ -12,23 +12,24 @@ const phaseGeneratePermutations = require('./lib/phase/permutation');
  * Parses a yaml file
  * @method parseYaml
  * @param  {String}  yaml Raw yaml
- * @return {Promise}      resoves POJO containing yaml data
+ * @return {Promise}      Resolves POJO containing yaml data
  */
 const parseYaml = yaml => (new Promise(resolve => resolve(Yaml.safeLoad(yaml))));
 
 /**
  * Parse the configuration from a screwdriver.yaml
  * @method configParser
  * @param  {String}   yaml      Contents of screwdriver.yaml
- * @param  {Function} callback  Function to call when done (error, { workflow, jobs })
+ * @param  {String}   apiUrl    API Url to retrieve templates (default to https://api.screwdriver.cd)
```

Comment:
How else can we look up templates? Intially I was thinking passing in a templateFactory, but that would break the ability to run this as command line. Discussed with @petey and decided to go with making API calls in here. What other options are you thinking?